src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestTrustedMasterKeyPaths.cs

```diff
@@ -0,0 +1,294 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted
+{
+    public class TestTrustedMasterKeyPaths : IClassFixture<SQLSetupStrategy>
+    {
+        private SQLSetupStrategy fixture;
+
+        static private string defaultConnectionString;
+
+        private readonly string tableName;
+
+        private readonly string columnMasterKeyPath;
+
+        public TestTrustedMasterKeyPaths(SQLSetupStrategy fixture)
+        {
+            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
+            builder.ConnectTimeout = 65534;
```

Comment:
Put comments for the magic number used here, if intentional, or reduce timeout to avoid hang.

Reply:
I actually don't know why the timeout is set to 65534. This number is widely used in TCE test suite and I just used the same number. There were no comments there either. Same value used in SqlCommandShould test cases as well.
Do you want me to use a smaller number?

Reply:
Yeah, if no purpose is defined, and if you know this connection should succeed, it should not timeout for so long (10 seconds should be good).

In cases where servers are turned off due to any reason, the tests would hang unnecessarily.